types/campaigns.ts

```diff
@@ -73,10 +73,25 @@ export type ConversionAudience = z.infer<typeof ConversionAudienceSchema>;
  * Asymmetric Reward Schema
  */
 export const AsymmetricRewardSchema = z.object({
+  /**
+   * The token address to reward
+   */
   tokenAddress: z.string(),
+  /**
+   * The amount to reward
+   */
   amount: z.string(),
+  /**
+   * Leave blank: The user public key that will be populated in the payout transaction
+   */
   userPubKey: z.string().nullish(),
+  /**
+   * Who can participate in the raffle, users, publishers, or both
+   */
   participants: z.nativeEnum(RaffleParticipants).nullish(),
+  /**
+   * Leave blank: The transaction hash of the payout transaction
+   */
   payoutTx: z.string().nullish(),
```

Comment:
in the PR i made today I remove thesed fields, the are not needed cuz i think this schema is only used in campaign creation